build.gradle

```diff
@@ -25,8 +25,15 @@ dependencies {
 	implementation "org.apache.cxf:cxf-spring-boot-starter-jaxws:3.4.2"
 	implementation "org.springframework.boot:spring-boot-starter-validation"
 
+	implementation "javax.activation:activation:1.1.1"
+	implementation "javax.xml.bind:jaxb-api:2.3.1"
```

Comment:
As Java 11 does not contain the required JAXB classes by default, we have to manually specify them as a dependency